learning.md

```diff
@@ -0,0 +1,24 @@
+## Learning and knowledge
+
+### Resources
+* We have a [wiki](https://sites.google.com/pagopa.it/io-wiki/io-home).
+
+### Internal workshops
+We periodically run short internal workshop sessions to facilitate knowledge sharing between between us.
+Subject may vary: they can be about explaining some new domain concepts in IO as well as share with other the use of a specific technology or introducing new features in our common tools.
+
+In few words: anything you think your colleagues can benefit from.
+ 
+
+#### Instructions
+###### before
+* Take the initiative: propose yourself to run a workshop. Alternatively, ask a colleague of yours to do that if you think she/he knows something worth sharing. 
+* Once scheduled, create a public calendar event and share it.
```

Comment:
- post the link to the calendar on #team_tech channel
- save all assets and sources (excalidraw, gdoc, slides, ...) and share them aside the video

Reply:
> post the link to the event on #team_tech channel

ok


> save all assets and sources (excalidraw, gdoc, slides, ...) and share them aside the video

it's said in the _after_ section